ProxyAppsSuite/tests/miniamr.yaml

```
@@ -0,0 +1,896 @@
+_base:
+   summary: 'Apply a stencil calculation on a unit cube computational domain'
+   build:
+      summary: building the miniAMR test
+      modules: [ gcc, openmpi ]
+      env:
+         CC: mpicc
+      cmds:
+          - git clone https://github.com/Mantevo/miniAMR
+          - cd miniAMR/ref 
+          - make || exit 1
+   scheduler: slurm
+   subtitle: "{{test_input.id}}"
+   run:
+       timeout: 1200
+       modules: [ gcc, openmpi ]
+   result_parse:
+       regex:
+           result:
+               regex: '================\s+Start\s+report\s+==================='
```

I think we should change this to match the "End Report" line at the bottom. There may be some cases where the test starts out successfully and prints out "Start report" but somehow "breaks"/"quits" in the middle of a test run and doesn't actually finish

```suggestion
               regex:  *End report*
```